ui/src/components/inputs/EditorWrapper.vue

```diff
@@ -122,6 +124,22 @@
         store.dispatch("plugin/updateDocumentation", {event,task});
     };
 
+    const flowParsed = computed(() => store.getters["flow/flowParsed"]);
+    const save = async () => {
+        const result = await store.dispatch("flow/save", {content: editorDomElement.value.$refs.monacoEditor.value})
+        if(result === "redirect_to_update"){
+            await router.push({
+                name: "flows/update",
+                params: {
+                    id: flowParsed.value.id,
+                    namespace: flowParsed.value.namespace,
+                    tab: "edit",
+                    tenant: router.currentRoute.value.params.tenant,
```

Comment:
why not use `useRoute()` instead of `useRouter()` to get the route?

Reply:
No other reason than copying the logic form `EditorView.vue` component. Amended now, as it totally makes sense.